sherlock/internal/oidc_models/auth_request.go

```diff
@@ -0,0 +1,105 @@
+package oidc_models
+
+import (
+	"database/sql"
+	"github.com/broadinstitute/sherlock/go-shared/pkg/utils"
+	"github.com/broadinstitute/sherlock/sherlock/internal/models"
+	"github.com/google/uuid"
+	"github.com/zitadel/oidc/v3/pkg/oidc"
+	"github.com/zitadel/oidc/v3/pkg/op"
+	"time"
+)
+
+// "AuthRequest implements the op.AuthRequest interface"
+var _ op.AuthRequest = &AuthRequest{}
```

Comment:
I like this pattern is an easily visible compile time validation

Reply:
Yeah it's convenient because if you write this as the first step GoLand will offer to generate stubs for all the methods